apps/github-cascading-app/src/cascading/cascading-probot.ts

```diff
@@ -56,8 +56,8 @@ export class CascadingProbot extends Cascading {
       this.logger.warn(`Several configuration files have been found (${configFileValidResponses.map((c) => c?.path).join(', ')}). The files will be ignored.`);
     } else if (configFileResponse) {
       try {
-        const configFileResponseWithContent: { content?: string; encoding?: 'utf8' | 'utf-8' | 'base64' } = configFileResponse as unknown as any;
-        const parsedConfig = configFileResponseWithContent.content && JSON.parse(Buffer.from(configFileResponseWithContent.content, configFileResponseWithContent.encoding || 'base64').toString());
+        const configFileResponseWithContent = configFileResponse as { content?: string; encoding?: 'utf8' | 'base64' };
+        const parsedConfig = JSON.parse(Buffer.from(configFileResponseWithContent.content ?? '{}', configFileResponseWithContent.encoding || 'base64').toString()) as Partial<CascadingConfiguration>;
```

Comment:
```suggestion
        const parsedConfig = JSON.parse(Buffer.from(configFileResponseWithContent.content || '{}', configFileResponseWithContent.encoding || 'base64').toString()) as Partial<CascadingConfiguration>;
```
passing empty string to `JSON.parse` will throw an exception